lib/dialogs/import/dropzone/index.tsx

```diff
@@ -63,14 +63,21 @@ function ImporterDropzone({
     onDrop,
   });
 
-  const DropzonePlaceholder = () => (
-    <Fragment>
-      {errorMessage.length > 0 ? <WarningIcon /> : <CloudIcon />}
-      {isDragActive
-        ? 'Drop files here'
-        : 'Drag and drop to upload files, or click to choose'}
-    </Fragment>
-  );
+  const DropzonePlaceholder = () => {
+    const instructions = isDragActive ? (
+      'Drop files here'
+    ) : (
+      <div className="drop-instructions">
+        Drag and drop to to select files or <span>browse</span> to choose
+      </div>
+    );
+    return (
+      <Fragment>
+        {errorMessage.length > 0 ? <WarningIcon /> : <CloudIcon />}
+        {instructions}
```

Comment:
personally I don't find it helpful to split this into a separate variable. did you extract it here because of the new `<div>`?

Reply:
Sort of. I was struggling to get HTML, the `div` and `span`, to render without extracting it.

Reply:
it should work if you inline it all, it's okay to mix strings and React components

```ts
{isDragActive ?
	'Drop files here'
: (
	<div className="drop-instructions">
		Drag and drop to select files or <span>browse</span> to choose
	</div>
)}
```

also I just noticed a typo "Drag and drop **to to** select files"

Reply:
Thank you on both of these points. I've fixed them up.